### DISCUSSION TITLE
Team discussions are not given in the search results.

### DISCUSSION THREAD

**Discussion by volkantash** [community/community#8345] [Discussions]
Selâm (Hi).

Arama sonuçlarında takım tartışmaları verilmiyor. (Team discussions are not given in the search results.)
![resim](https://user-images.githubusercontent.com/25375593/144447089-03b6da5c-1d8e-4668-a451-a3fd0171ef3f.png)

Ulaşmak istenilen başlık. (The title you want to reach.)
![resim](https://user-images.githubusercontent.com/25375593/144447221-3f252603-8c37-4d30-809f-4dcac96314e6.png)

**Comment by jsoref** [comment]
Team discussions are unfortunately an unrelated feature.

**Reply by volkantash** [reply]
It was an old request. Team discussions have been removed from Github. No need anymore.
